src/common/HTTPRequest.ts

```diff
@@ -295,12 +382,42 @@ export class HTTPRequest {
    * ```
    *
    * @param overrides - optional overrides to apply to the request.
+   * @param priority - (optional, default 0). Priority 0 will execute immediately.
```

Comment:
In the current implementation, `0` is not implemented as an immediate execution (only `undefined`)